pyrasterframes/src/main/python/docs/nodata-handling.pymd

```diff
@@ -132,29 +135,92 @@ masked.select(rf_no_data_cells('blue_masked'), rf_tile_sum('mask')).show(10)
 
 It's also nice to view a sample.
 
-```python show_masked
+```python
 sample = masked.orderBy(-rf_no_data_cells('blue_masked')).select(rf_tile('blue_masked'), rf_tile('scl')).first()
 display(sample[0])
 ```
 
 And the original SCL data.
 
-```python show_scl
+```python
 display(sample[1])
 ```
 
-## NoData in Arithmetic Operations
+## Changing a Tile's NoData Values
+
+One way to mask a tile is to make a new tile with a user defined NoData value. We will explore this method below. First, lets create a rasterframe from a tile with values of 0, 1, 2, and 3.
+
+```python create_dummy_tile
+tile_size = 100
```

Comment:
Idea: If you make these smaller (say, 3 or 4) you could use the `rf_render_martix` function to show the contents of the tile within the dataframe.

Reply:
If you think thank is preferable I can do that. I personally like the rendered image.

Reply:
![image](https://user-images.githubusercontent.com/8228717/62328339-e1e1dc80-b480-11e9-8a90-fa2409589da4.png)
You are suggesting something like this? I don't know how to deal with the ascii.

Reply:
Nvm I didn't know you can set truncate to false. I personally still like the image but I'm open to whatever.

Reply:
Stick with the image... I wasn't paying attention and didn't think that `display` was rendering the tile, which i think is better in this case.